src/components/Profile.vue

```diff
@@ -1,7 +1,7 @@
 <template id="profile">
 
     <div class="profile-bar">
-        <div>Acting for <router-link v-bind:to="'/user'">P &amp; J Millward</router-link></div>
+        <div>Acting for <router-link v-bind:to="'/user'"><strong>P &amp; J Millward</strong></router-link></div>
         <div v-if="recVouchers[0] && recVouchers[0].length > 0"><strong>{{ recVouchers[0].length }}</strong> vouchers waiting</div>
         <div>Managing <router-link v-bind:to="'/user'">{{ selectedTrader.name }}</router-link></div>
```

Comment:
Uh... I think this has happened because it asked me if I wanted to merge 0.2/release into my branch.